mapscript/swiginc/reprojection.i

```diff
@@ -0,0 +1,45 @@
+/* ===========================================================================
+   Project:  MapServer
+   Purpose:  SWIG interface file for mapscript reprojectionObj extensions
+   Author:   Steve Lime 
+             Even Rouault
+             
+   ===========================================================================
+   Copyright (c) 1996-2022 Regents of the University of Minnesota.
+   
+   Permission is hereby granted, free of charge, to any person obtaining a
+   copy of this software and associated documentation files (the "Software"),
+   to deal in the Software without restriction, including without limitation
+   the rights to use, copy, modify, merge, publish, distribute, sublicense,
+   and/or sell copies of the Software, and to permit persons to whom the
+   Software is furnished to do so, subject to the following conditions:
+ 
+   The above copyright notice and this permission notice shall be included
+   in all copies or substantial portions of the Software.
+ 
+   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+   OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+   THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+   FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+   DEALINGS IN THE SOFTWARE.
+   ===========================================================================
+*/
+
+
+%extend reprojectionObj
+{
+    /// Create new instance of reprojectionObj. Input parameters are two projectionsObj's - in and out.
+    reprojectionObj(projectionObj *in, projectionObj *out) 
+    {
+	if(!in || !out) return NULL;
```

Comment:
indentation issue.
and msProjectCreateReprojector() supports one of in or out to be NULL, and that means that it is the geographic CRS corresponding to the non-NULL argument. So the test should rather use a &&

Reply:
Made that change, thanks!